tutorials/ner_multiple_trials/project.yml

```diff
@@ -0,0 +1,119 @@
+  - dest: "assets/raw-en-wikineural-train.iob"
+    description: "WikiNEuRal (en) training dataset"
+    url: https://raw.githubusercontent.com/Babelscape/wikineural/master/data/wikineural/en/train.conllu
+  - dest: "assets/raw-en-wikineural-dev.iob"
+    description: "WikiNEuRal (en) dev dataset"
+    url: https://raw.githubusercontent.com/Babelscape/wikineural/master/data/wikineural/en/val.conllu
+  - dest: "assets/raw-en-wikineural-test.iob"
+    description: "WikiNEuRal (en) test dataset"
+    url: https://raw.githubusercontent.com/Babelscape/wikineural/master/data/wikineural/en/test.conllu
+
+commands:
+  - name: "preprocess"
+    help: "Preprocess the WikiNEuRal dataset to remove indices and update delimiters."
+    script:
+      - python -m scripts.preprocess assets/raw-en-wikineural-train.iob assets/en-wikineural-train.iob
+      - python -m scripts.preprocess assets/raw-en-wikineural-dev.iob assets/en-wikineural-dev.iob
+      - python -m scripts.preprocess assets/raw-en-wikineural-test.iob assets/en-wikineural-test.iob
+    deps:
+      - assets/raw-en-wikineural-train.iob
+      - assets/raw-en-wikineural-dev.iob
+      - assets/raw-en-wikineural-test.iob
+    outputs:
+      - assets/en-wikineural-train.iob
+      - assets/en-wikineural-dev.iob
+      - assets/en-wikineural-test.iob
+
+  - name: "convert"
+    help: "Convert IOB dataset into the spaCy format."
+    script:
+      - python -m spacy convert assets/en-wikineural-train.iob corpus/ -n 10
+      - python -m spacy convert assets/en-wikineural-dev.iob corpus/ -n 10
+      - python -m spacy convert assets/en-wikineural-test.iob corpus/ -n 10
+    deps:
+      - assets/en-wikineural-train.iob
+      - assets/en-wikineural-dev.iob
+      - assets/en-wikineural-test.iob
+    outputs:
+      - corpus/en-wikineural-train.spacy
+      - corpus/en-wikineural-dev.spacy
+      - corpus/en-wikineural-test.spacy
+
+  - name: "train"
+    help: "Train a named-entity recognition (NER) model for a multiple number of trials."
+    script:
+      - >-
+        python -m scripts.train_with_trials
+        configs/${vars.config}
+        --n-trials ${vars.trials}
+        --nlp.lang en
+        --output training/
+        --paths.train corpus/en-wikineural-train.spacy
+        --corpora.train.limit ${vars.limit}
+        --paths.dev corpus/en-wikineural-dev.spacy
+        --corpora.dev.limit ${vars.limit}
+        --training.max_steps ${vars.max_steps}
+        --gpu-id ${vars.gpu_id}
+    deps:
+      - corpus/en-wikineural-train.spacy
+      - corpus/en-wikineural-dev.spacy
+    outputs:
+      - training
+
+  - name: "evaluate"
+    help: "Evaluate all models for each trial, then summarize the results."
+    script:
+      - >-
+        python -m scripts.evaluate_with_trials
+        training/
+        corpus/en-wikineural-test.spacy
+        --output-dir metrics/
+        --gpu-id ${vars.gpu_id}
+
+  - name: "clean"
+    help: "Remove cached files"
+    script:
+      - rm -rf training/
```

Comment:
clear out also `corpus` and `metrics`?

Reply:
Updated in https://github.com/explosion/projects/pull/123/commits/ac1b600009bf860a7b17ca8c53527d9066f334c6